src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+    Red,
+    Yellow,
+    Blue,
+    Green,
+    Pink,
+    Purple,
+    White
```

Comment:
Enum values should be in uppercase as per Java naming conventions. Please refer to the checklist item about naming constants and enum values.